Extend troubleshooting (#196)
greenbone/docs: src/common/container/troubleshooting.md

already initialized, all tables, users, permissions and extensions exist,
errors are raised. At the moment, it is not possible to silence these errors but
they can be ignored safely.

### `osp_scanner_feed_version: failed to connect to /run/ospd/ospd-openvas.sock`

> If the `gvmd` log contains this error `gvmd` is not able to connect to the

Reviewer: ```suggestion
If the `gvmd` log contains this error, `gvmd` is not able to connect to the
```

> `ospd-openvas` scanner. Most likely `ospd-openvas` isn't running. Try to restart

Reviewer: ```suggestion
`ospd-openvas` scanner. Most likely `ospd-openvas` is not running. Try to restart
```

the corresponding container with:

```{code-block} shell
---
caption: Restart the ospd-openvas scanner
---
docker-compose -f $DOWNLOAD_DIR/docker-compose.yml -p greenbone-community-edition \
    restart ospd-openvas
```

> If you still get errors you need to take a look at the `ospd-openvas` logs.

Reviewer: ```suggestion
If you still get errors, you need to take a look at the `ospd-openvas` logs.
```
